### DISCUSSION TITLE
问下 你是这个石杉笔记的作者吗 | The relationship between yanglbme and shishan

### DISCUSSION THREAD

**Discussion by goodkkkkk** [doocs/advanced-java#45] [Q&A] [answered]
为什么这么少内容会 这么多赞 怎么做到的

**Comment by 1510963833** [comment]
不讲究量，要的是质

**Comment by goodkkkkk** [comment]
没毛病 感觉 还是公众号的好点 不知道他俩啥关系

**Comment by yanglbme** [comment, marked as answer]
@goodkkkkk 同学你好，本人不是石杉，这点先给你一个明确的回答。

- 公众号（石杉的架构笔记，ID: shishan100）里的文章，基本是每日推送一篇的频率，部分读者可能很难站在全局的角度，系统梳理好知识点之间的关系。如果你觉得公众号好一点，保持对它的关注即可。
- 我整理了这一系列的知识，希望能给读者们更加系统、完整地梳理 Java 的进阶知识体系，我相信对很多读者来说，都会有所收获。
- 另外，目前不少技术类公众号都在商业互吹，各种广告信息，很大程度上已经弱化了其原有的特色。在这个项目里，绝对不会为了一些利益而影响到用户的阅读学习体验。

每个人都有自己的偏好，我很尊重每一位读者的选择。

最后，要谢谢 @1510963833 同学的肯定。

**Comment by yanglbme** [comment]
一些同学似乎来针对我了，说这个仓库的目的性太强。

这个仓库的目的并不是想让读者把这些知识点背下来，从而更好应付面试。要记住，IT 知识从来都不是死记硬背。希望读者能通过这个仓库，对日常工作生活有一个更深的思考，别老大说用就用，却不知道为什么要用，有没有更好的解决方案，背后的原理是什么。

同时，让读者更清楚面试官问问题的意图是什么。仓库中**面试官心理分析**这一块，真的值得各位读者看一看。我想，对于不少读者来说，很多时候知识点都会，但在面试时却 GET 不到面试官的点。这个仓库希望能给读者一些启发。读懂面试官问问题的意图，在面试时可能会更加游刃有余，从而取得一个较好的结果。

另外，也是希望能够全面为读者梳理分布式、高并发、高可用等领域所涉及的知识。站在一个全局的视角，我想，学习效果会更好的。

**Comment by dingguotao** [comment]
感谢作者用心花时间来记录笔记，@yanglbme ，不用理会他们。众口难调，不用理会那些喷子。

**Comment by yanglbme** [comment]
@iclouding 嗯嗯 好的，谢谢你的支持 :)

**Comment by DeanWanghewei** [comment]
@yanglbme 感谢作者的用心

**Comment by yanglbme** [comment]
@DeanWanghewei Thank you :)

**Comment by wxxy20071547** [comment]
这些人真是什么都能喷啊，你要关注公众号去关注好了，人家做了总结分享还逼逼， 哎

**Comment by JavaJack86** [comment]
希望有netty和nginx相关的文章哦~

**Comment by yanglbme** [comment]
@wxxy20071547 谢谢同学的支持。我一直相信，只要自己认真做好，一定会被大家所接受。

另外，开源项目维护者有自己的风格，读者有自己的偏好，选择自己觉得好的即可。

**Comment by yanglbme** [comment]
@JavaJack86 目前一人利用业余时间维护 @doocs @TheAlgorithms 多个开源组织项目，时间精力都有限，我之后抽空更新。

如果可以，我很希望读者们可以参与进来，分享自己对知识的理解。

**Comment by JavaJack86** [comment]
嗯 不着急 谢谢啦

**Comment by Robert-JQ** [comment]
写的太好了，我可以引用转载到我的语雀知识库吗@yanglbme

**Comment by yanglbme** [comment]
@Robert-JQ 可以的，但请注明出处。

**Comment by Robert-JQ** [comment]
> @Robert-JQ 可以的，但请注明出处。

ok

**Comment by huangjingyes** [comment]
感谢作者，给我们带来很多启发，更多去关注一些深入的问题，而不是只会用

**Comment by cherryxyz** [comment]
希望大家可以分享一些心得体会，工作学习经验等能让我这种菜鸟能少走一些弯路~加油

**Comment by yanglbme** [comment]
@cherryxyz 我也一直希望可以有更多朋友参与进来。

这是一个社会化的时代。即便是那些平时与 code 打交道较多的 IT 技术人员，也需要多与他人分享交流，在交流中认清自己，提升能力，与他人共同进步。

我目前也在考虑哪种分享形式比较好。大家提 issue，我整理成文，还是直接开一个专门的 issue 来交流？如果直接在 issue 上交流的话，会显得有点乱。或是其它形式？读者们若有好的想法，都可以分享一下。

**Comment by yanglbme** [comment]
@huangjingyes 踏踏实实学习新东西，平时多思考为什么，我们一起努力、一起进步。

**Comment by wxxy20071547** [comment]
哈哈  石衫公众号给推荐了你的github

**Comment by yanglbme** [comment]
@wxxy20071547 哈哈，难怪感觉比平常多了一些关注。要感谢一下公众号，让更多朋友可以看到这些用心整理的干货文章~

**Comment by 1302cb** [comment]
哈哈哈。没事就去刷刷掘金，然后关注了石杉笔记。后面偶然看到推荐的GitHub。帮助很大。已经顺利收到offer。。感谢作者

**Comment by yanglbme** [comment]
@1302cb 恭喜，很棒很棒。

**Comment by InvokerPeng** [comment]
写的很好，不过针对你之前的微服务还未详细写的，想看到，求他针对你的帖子之类的，不要去管，你写的是真的好

**Comment by evan009** [comment]
恩恩，只要表名文章来源就好，说的一点特别好，现在很多公众号都是商业互吹

**Comment by dfohyes** [comment]
写的太好了，收获很多。顺便问下，有数据中台、数据平台、数据仓库、数据分析等大数据相关的知识吗

**Comment by yanglbme** [comment]
@dfohyes 暂时没有

**Comment by trennble** [comment]
虽然很多人问了，但是我还是问一下，因为这个repo和石杉的这个repo是一样的https://github.com/shishan100/Java-Interview-Advanced 这里我不是要说作者抄石杉，因为我看了下作者热commit，比石杉的repo要早很多，然后如果是石杉抄作者的repo我建议作者还是维护一下自己，因为石杉还就这个repo发了付费视频课程的

**Comment by RocketTH** [comment]
> 虽然很多人问了，但是我还是问一下，因为这个repo和石杉的这个repo是一样的https://github.com/shishan100/Java-Interview-Advanced 这里我不是要说作者抄石杉，因为我看了下作者热commit，比石杉的repo要早很多，然后如果是石杉抄作者的repo我建议作者还是维护一下自己，因为石杉还就这个repo发了付费视频课程的

这些内容基本都是中华石衫原创，何来clone？主人来了，维护啥维护啊

**Comment by RocketTH** [comment]
版权归中华石衫所有，作者如果感觉不公平，可以写一篇公众号发文，或者艾特中华石衫，我可以帮你转发

> 写的太好了，收获很多。顺便问下，有数据中台、数据平台、数据仓库、数据分析等大数据相关的知识吗

等中华石衫原创出了，这里就会有了

**Comment by swinghigh** [comment]
很棒！

**Comment by blackwhitechen** [comment]
东西好不好不是你一句话就能否定，离不开那些愿意分享的人，离不开开源的情怀@yanglbme   谢谢你，杨哥
